pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -207,9 +213,11 @@ func NewFit(_ context.Context, plArgs runtime.Object, h framework.Handle, fts fe
 //	    Memory: 1G
 //
 // Result: CPU: 3, Memory: 3G
-func computePodResourceRequest(pod *v1.Pod) *preFilterState {
+func computePodResourceRequest(pod *v1.Pod, opts ResourceRequestsOptions) *preFilterState {
```

Comment:
I'm make this and other function a method of the Fit plugin, but feel free to leave todo here for follow up instead.

Reply:
Left a TODO

Reply:
I meant to make it a method, so to get rid of the opts param. Not sure if I understand the TODO

```suggestion
func (f *Fit) computePodResourceRequest(pod *v1.Pod) *preFilterState {
	// pod hasn't scheduled yet so we don't need to worry about InPlacePodVerticalScalingEnabled
	reqs := resource.PodRequests(pod, resource.PodResourcesOptions{
		PodLevelResourcesEnabled: f.enablePodLevelResources,
	})
	result := &preFilterState{}
	result.SetMaxResource(reqs)
	return result
```

Reply:
Yeah, I'm not sure if it's possible either, as this function is used in a few other places... so feel free to ignore my comment.